src/desktop/apps/auction_reaction/server.tsx

```diff
@@ -1,8 +1,9 @@
 import express from "express"
-import { bidderRegistration, auctionFAQRoute } from "./routes"
+import { bidderRegistration, auctionFAQRoute, confirmBidRoute } from "./routes"
 
 export const app = express()
 
 app.get("/auction-faq", auctionFAQRoute)
 app.get("/auction-registration/:auctionID", bidderRegistration)
 app.get("/auction-registration2/:auctionID*", bidderRegistration)
+app.get("/auction/:auctionID/bid2/:artworkID*", confirmBidRoute)
```

Comment:
I think we can remove the trailing asterisk here. I don't think we're planning to support any nested routes.